### DISCUSSION TITLE
Using reverse proxy with path does not work with basePath

### DISCUSSION THREAD

**Discussion by bkeetman** [vercel/next.js#16588] [Help] [answered]
We are trying to run a website on `account/product`.

```
const nextConfig = {
    basePath: '/account/product'
};
```

And in this case using Traefik's reverse proxy to point the url to:
```
- 'traefik.http.routers.product-frontend.rule=Host(`www.website.nl`) && Path(`/account/product`)'
```

Visiting the `/account/product` url gives a `200` but all the assets in `/account/product/_next/*` keep giving `404` errors.

What are we doing wrong?

Extra info:
* We need to point this application using `Path("/account/product")` because other applications are running on `/`.

**Comment by edwinboonold** [comment, marked as answer]
We solved the issue by using `PathPrefix` instead of `Path`

`- 'traefik.http.routers.product-frontend.rule=Host(`www.website.nl`) && PathPrefix(`/account/product`)'

This solved our issues at least for local development
